applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -1608,12 +1481,18 @@ int unicast_client_send(struct le_audio_encoded_audio enc_audio)
 	int ret;
 #if CONFIG_BT_AUDIO_TX
 	struct bt_bap_stream *bap_tx_streams[CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT];
+	uint8_t channel_mask[CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT] = {0};
 
 	for (int i = 0; i < CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT; i++) {
 		bap_tx_streams[i] = &headsets[i].sink_stream;
+		if (headsets[i].location == BT_AUDIO_LOCATION_FRONT_RIGHT) {
```

Comment:
Maybe we could have a comment in here for reminding the user that "mono" and "left" are both allocated to AUDIO_CH_L.